src/SuggestionsOverlay.tsx

```diff
@@ -46,6 +46,9 @@ interface SuggestionsOverlayProps<T extends BaseSuggestionData> {
 
     /** Callback invoked on mouse down in the suggestions overlay. */
     onMouseDown: () => void;
+    slotProps?: {
+        root?: Omit<PopperProps, 'anchorEl' | 'open'>;
```

Comment:
Let's call this `popper` instead of root. I could imagine we might also add `paper` and `list` fields here in the future, and it will be more obvious what each refers to if they exactly match the name of the component.